src/documents/tutorial-prefsFramework/CreatingAPrefsEditor.md

```diff
@@ -0,0 +1,779 @@
+
+Let’s look at the Schema itself in detail:
+```javascript
+auxiliarySchema: {
+    // the loaderGrade identifies the "base" form of preference editor desired
+    loaderGrades: ["fluid.prefs.fullNoPreview"],
+
+    // 'terms' are strings that can be re-used elsewhere in this schema;
+    terms: {
+        templatePrefix: "html"
+    },
+
+    // the main template for the preference editor itself
+    template: "%templatePrefix/minEditor.html",
+
+    autoPilot: {
+        // this 'type' must match the name of the pref in the primary schema
+        type: "minEditor.autoPilot",
+        panel: {
+            // this 'type' must match the name of the panel grade created for this pref
+            type: "minEditor.panels.autoPilot",
+
+            // selector indicating where, in the main template, to place this panel
+            container: ".mec-autoPilot",
+
+            // the template for this panel
+            template: "%templatePrefix/autoPilot.html"
+        }
+    }
+}
+```
+An auxiliary schema can be generally divided into two types of properties:
+
+1. top-level members, defining globally-used values, and
+2. per-preference members (one per preference), defining the specific requirements for each preference.
+
+
+##### Loader Grade #####
+The _loader grade_ specifies the _type_ of Preference Editor:
+
+```javascript
+    loaderGrades: ["fluid.prefs.fullNoPreview"],
+```
+
+The Preference Framework provides three pre-defined types of Editor:
+1. separated panel (the default): This is a page-width panel collapsed at the top of the page;
+it slides down when activated by the user.
+2. full page, no preview: This is a Preference Editor that occupies the full page.
+3. full page, with preview: This is a Preference Editor that occupies the full page, but includes
+provisions for an iframe in the page to preview any changes made by the Editor.
+
+In the code snippet above, the `loaderGrades` option is used to specify the “full page, no preview” form.
+
+##### Templates #####
+The Auxiliary Schema must declare where to find the main HTML template for the Preference Editor.
+In our example, this template is located in the same folder as other HTML templates.
+The Auxiliary Schema allows you to define `terms` – strings that can be re-used elsewhere in the
+schema. Here, it is being used to define, in a single place, the path to where the HTML templates are:
+
+```javascript
+    terms: {
+        templatePrefix: "html"
+    },
+```
+
+The template property specifies the main HTML template for the entire Preference Editor:
+
+```javascript
+    template: "%templatePrefix/minEditor.html",
+```
+You can see the full text of this file, `minEditor.html`, in the github repo:
+https://github.com/fluid-project/infusion/tree/master/examples/framework/preferences/minimalEditor/html/minEditor.html
+The main thing to note in the template is the placeholder for the Panel,
+in this example a `<div>` with the class `mec-autoPilot”`:
```

Comment:
I wonder if you've missed pushing up some changes, as this is `mec-heatedSeats"` on your github pages instance.http://acheetham.github.io/infusion-docs/infusion/development/tutorial-prefsFramework/CreatingAPrefsEditor.html#templates